### PR TITLE
Show errors for map form hidden fields

### LINKED ISSUE #1496: Display plan form error of hidden fields
In cases were there is an error on a hidden field the plan form submit button remains disabled without any warning.
User should be able to see the error blocking them from submitting a plan.

### PR DESCRIPTION
**Pull request by ciremusyoka**
Closes #1496 
When there is an error in one of the hidden fields the plan form submit button is disabled and a user can't know why. This pr will show those errors at the bottom of the page just before the submit button.

### CODE REVIEW

**Comment by moshthepitt** [comment]
@ciremusyoka could you include a screenshot please?

**Comment by ciremusyoka** [comment]
Screenshot
Screenshot from 2021-07-14 10-24-04

cc @moshthepitt

File: src/components/forms/PlanForm/index.tsx [hunk 1]
```diff
@@ -1247,6 +1249,14 @@ const PlanForm = (props: PlanFormProps) => {
                 </ModalBody>
               </Modal>
             )}
+            {Object.keys(errors).map(
+              (key, i) =>
+                defaultHiddenFields.includes(key) && (
+                  <small key={i} className="form-text text-danger">
```
[the remark this answers is not in the document]
Comment (ciremusyoka) [reply src/components/forms/PlanForm/index.tsx:1255]: Okay let me try it out

[the remark this answers is not in the document]
Comment (ciremusyoka) [reply src/components/forms/PlanForm/index.tsx]: Screenshot from 2021-07-14 11-56-13

[the remark this answers is not in the document]
Comment (ciremusyoka) [reply src/components/forms/PlanForm/index.tsx]: Screenshot from 2021-07-15 17-14-33

**Review by moshthepitt — APPROVED** [review]

**Review by KipSigei — APPROVED** [review]

**Pull request merged**